lr-cli/cortx_setup/commands/storageset/config/durability.py

```diff
@@ -76,34 +74,35 @@ def run(self, **kwargs):
                 f'json://{CONFSTORE_CLUSTER_FILE}'
             )
 
-            ss_name = Conf.get(index, f'cluster>{cluster_id}>storage_set[0]>name')
-
-            durability_dict = {durability_type: {}}
+            ss_name = Conf.get(index,
```

Comment:
How will this work for multiple storage sets?
Tomorrow we may have 2-3 storage sets in which case only storage_set[0] will not work

Reply:
Lets leave storage-set for now